graspologic/match/gmp.py

```diff
@@ -178,6 +179,12 @@ def fit(self, A, B, seeds_A=[], seeds_B=[]):
             An array where each entry is an index of a node in `B` The elements of
             ``seeds_A`` and ``seeds_B`` are vertices which are known to be matched, that is,
             ``seeds_A[i]`` is matched to vertex ``seeds_B[i]``.
+        S : 2d-array, square
+            A square similarity matrix. Should be same shape as ``A`` and ``B``.
```

Comment:
what does this mean when A and B have different shape? Can't it be N_A by N_B?

Reply:
yeah you're right, should be n_a x n_b